crates/iceberg-catalog/src/config.rs

```diff
@@ -80,6 +81,12 @@ pub struct DynAppConfig {
     /// reverse proxy before routing to the catalog service.
     /// Example value: `{warehouse_id}`
     prefix_template: String,
+    /// CORS allowed origins. If not set, CORS is disabled.
+    #[serde(
+        deserialize_with = "deserialize_origin",
+        serialize_with = "serialize_origin"
+    )]
+    pub allow_origin: Option<Vec<HeaderValue>>,
```

Comment:
As there is more than one, should we name it allowed_origins?

Reply:
Do we ever need serialization of the config? can we get rid of those functions?

Regarding name - maybe allow_origin is OK as this is also how tower calls it.

Reply:
Iirc figment needs it